src/main/java/org/assertj/core/internal/Objects.java

```diff
@@ -932,6 +937,169 @@ public boolean areEqualToComparingOnlyGivenFields(Object actual, Object other,
     if (actual.hashCode() == other.hashCode()) throw failures.failure(info, shouldNotHaveSameHashCode(actual, other));
   }
 
+  /**
+   * Asserts that two <code>{@link Comparable}</code>s are equal by invoking
+   * <code>{@link Comparable#compareTo(Object)}</code>.<br>
+   *
+   * @param <T> used to guarantee that two objects of the same type are being compared against each other.
+   * @param info contains information about the assertion.
+   * @param actual the actual value.
+   * @param expected the expected value.
+   * @throws AssertionError if the actual value is {@code null}.
+   * @throws AssertionError if the actual value is not equal to the expected one. This method will throw a
+   *           {@code org.junit.ComparisonFailure} instead if JUnit is in the classpath and the expected and actual
```

Comment:
likely an opentest4j `AssertionFailedError` but I don't think it's important, probably better removing the whole `This method will throw a....`  sentence

Reply:
Same as the other comment, let's drop trivial javadoc for internal methods.